xchange-gdax/src/main/java/org/knowm/xchange/gdax/GDAXAdapters.java

```diff
@@ -232,23 +237,29 @@ public static Order adaptOrder(GDAXOrder order) {
 
   /** The status from the GDAXOrder object converted to xchange status */
   public static OrderStatus adaptOrderStatus(GDAXOrder order) {
-    if(order.getStatus().equals("done")) {
+
+    if(order.getStatus().equals("pending")) {
+      return OrderStatus.PENDING_NEW;
+    }
+
+    if(order.getStatus().equals("done") || order.getStatus().equals("settled")) {
 
       if(order.getDoneReason().equals("filled"))
         return OrderStatus.FILLED;
 
-      return null;
-
+      return null; // FIXME could be STOPPED perhaps. GDAX documentation doesn't really cover the possible
```

Comment:
Log an issue?

Reply:
maybe we should have an UNKNOWN status or similar.

Reply:
Yeah, that was my thinking. Happy to add that if you're comfortable with the API impact.